.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: node_js
+node_js:
+  - "8.9.4"   # Likely set Azure webapp minimum
```

Comment:
would suggest removing, and only testing with latest 8.x

Reply:
I'd prefer but I found too many of our own webapps defaulting to 8.9.4 and I'd assume others are doing that.  See https://github.com/hmcts/cnp-module-webapp/blob/2bd9e164e2ba9b701b41b0f0f11f3d2a684cbe70/variables.tf#L34

Reply:
pretty sure that's actually ignored, and isn't the one used at runtime.
It uses whats in the engines block of your package.json or latest available

Reply:
I had a clearer understanding when I actually built these PRs :) and I'm loath to retest tbh.  Happy to drop it if you are confident.

Reply:
Nope try bar-web, their engine is asking for `>=8.5.0` https://github.com/hmcts/bar-web/blob/master/package.json#L5-L7 they won't get the latest they're getting 8.9.4 because tht meets the criterion. So it is being used, just depends on what your engine demand is as to whether it will get ignored.  I'd suggest leaving it it and/or changing what we specify as the default/default for a webapp.

Reply:
Weird, could you bump the default to latest 8.x?

Reply:
i.e I don't think we should be testing 2 specific 8.x versions

Reply:
Yip let me PR a bump to 8.11.1 for Azure tomorrow. And then will relook at this.

Reply:
@timja Adjusted to drop `8.9.4`.  Now testing 4: lts (current and azure), latest (current and azure)